docs/usage_with_relay.md

```diff
@@ -0,0 +1,163 @@
+pg_graphql implements the [GraphQL Global Object Identification Specification](https://relay.dev/graphql/objectidentification.htm) (`Node` interface) and the [GraphQL Cursor Connections Specification](https://relay.dev/graphql/connections.htm#) to be compatible with [Relay](https://relay.dev/).
+
+## Relay Setup
+
+1. Follow the [Installation Guide](https://relay.dev/docs/getting-started/installation-and-setup/)
```

Comment:
I'd recommend changing these all to subheadings so that devs can "link" to the steps. eg:

```
###  Pre-requisites

Follow the [Installation Guide](https://relay.dev/docs/getting-started/installation-and-setup/).

### Configure Relay

Modify your `relay.config.js` file to reflect the following:
```

etc

Reply:
Agreed. It fixes the auto numbering issue too 🙌🏻